modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -734,11 +734,12 @@ private void _localizeNumericDDMFormFieldValues() {
 
 	private void _resetInvisibleFieldValue() {
 		_ddmFormFieldsPropertyChanges.forEach(
-			(fieldName, fieldProperty) -> {
+			(fieldContextKey, fieldProperties) -> {
```

Comment:
@linolaoj, please add the prefix `ddmFormField` to these parameters, e.g. rename to `ddmFormFieldContextKey` and `ddmFormFieldProperties`.

A few months ago, Brian raised his concern regarding the inconsistency of the DDM modules naming, there are codes using the `ddm` prefix and other codes not using it, by that time we agreed to always use the `ddm` prefix. Therefore, this is the reason I'm asking you to change it. 🙂